internal/artifacts/project.go

```diff
@@ -25,8 +26,13 @@ const (
 
 // ExecBuild - Execute MTA project build
 func ExecBuild(makefileTmp, source, target string, extensions []string, mode, mtar, platform string, strict bool, jobs int, outputSync bool, wdGetter func() (string, error), wdExec func([][]string, bool) error, useDefaultMbt bool) error {
+	message, err := version.GetVersionMessage()
+	if err == nil {
+		logs.Logger.Info(message)
+	}
+
```

Comment:
but if err!=nil?

Reply:
Then there's no point in printing the message... I don't think it should prevent the build.